package.json

```diff
@@ -41,15 +41,15 @@
     "jest-serializer-vue": "2.0.2",
     "lodash-es": "4.17.15",
     "lru-cache": "6.0.0",
-    "mini-css-extract-plugin": "github:brophdawg11/mini-css-extract-plugin#urbn",
+    "mini-css-extract-plugin": "1.3.0",
```

Comment:
Switch back to the normal package now that our required functionality is available.

The fork we were pointing to added an `insertInto` option to control the ordering of injected extracted CSS stylesheets.  This approach was subsequently replaced by a more flexible `insert` API so now we can switch back to the normal package and away from the fork.

See also https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/CHANGELOG.md#110-2020-10-19

Note we we apply the `cssInsert` `vue-ssr-build` option to the underlying `mini-css-extract-plugin` `insert` option: https://github.com/brophdawg11/vue-ssr-build/pull/82/files#diff-c2ab562cb8834a1a0f651353e7e8d264a26ea087d7f79e8e8a94f846212437f0R190